packages/ng/docs/demo/src/app/tooltip/basic/basic.html

```diff
@@ -0,0 +1,2 @@
+<span [luTooltip]="'LOL'">That tooltip</span><br>
```

Comment:
write `<span luTooltip="LOL" />`, thats the same thing but you dont have to use multiple quotes